internal/integration/provision/upgrade.go

```diff
@@ -70,18 +70,19 @@ type upgradeSpec struct {
 }
 
 const (
-	previousRelease = "v0.8.4"
-	stableRelease   = "v0.9.0-alpha.5"
+	previousRelease = "v0.8.5"
```

Comment:
need to update versions in `Makefile` `RELEASES` variable to match these (so that required artifacts are downloaded)